filecoindot-nft/src/lib.rs

```diff
@@ -0,0 +1,147 @@
+// Copyright 2021 ChainSafe Systems
+// SPDX-License-Identifier: LGPL-3.0-only
+
+//! # Filecoin Governance Pallet
```

Comment:
This should have a different doc-comment regarding NFT

Reply:
oops, you got me